src/Calendar.js

```diff
@@ -242,6 +242,18 @@ class Calendar extends React.Component {
    eventPropGetter: PropTypes.func,
 
    /**
+    * Optionally provide a function that returns an object of className or style props
+    * to be applied to the the time-slot node.
```

Comment:
can we add a note here advising caution? styles that change layout or position may break the calendar in unexpected ways.